web/client/components/background/BackgroundDialog.jsx

```diff
@@ -170,7 +170,11 @@ export default class BackgroundDialog extends React.Component {
                         onChange={event => this.setState({ format: event && event.value })}
                         value={this.state.format || this.props.defaultFormat}
                         clearable
-                        options={this.props.formatOptions}
+                        isLoading={!this.props.capabilities}
+                        options={
+                            (this.props.capabilities?.capabilities?.layerOptions?.imageFormats || this.props.formatOptions || [])
```

Comment:
double capabilities?

Reply:
It's because the layer capabilities contains another key called capabilities that represent the original resource
https://github.com/geosolutions-it/MapStore2/blob/master/web/client/utils/WMSUtils.js#L60